src/main/protobuf/build/buildfarm/v1test/buildfarm.proto

```diff
@@ -660,3 +660,59 @@ message BlobWriteKey {
 
   string identifier = 2;
 }
+
+message OperationTimesBetweenStages {
+  float queued_to_match = 1;
+
+  float match_to_input_fetch_start = 2;
+
+  float input_fetch_start_to_complete = 3;
+
+  float input_fetch_complete_to_execution_start = 4;
+
+  float execution_start_to_complete = 5;
+
+  float execution_complete_to_output_upload_start = 6;
+
+  float output_upload_start_to_complete = 7;
+
+  int32 number_of_operation = 8;
+
+  int32 period = 9;
+}
+
+message WorkerProfileMessage {
+  // number of Entry in CAS LRU
+  int64 CAS_entry_count = 1;
+
+  // number of Entry in CAS
+  int64 CAS_directory_entry_count = 2;
+
+  // sum up size of containingDirectories of all active entries
+  int64 entry_containing_directories_count = 3;
+
+  // the observed largest size of Entry.containingDirectories, this is the largest value
+  // since the start of the specific worker
+  int64 entry_containing_directories_max = 4;
+
+  // number of evicted Entries since last profile request
+  int32 CAS_evicted_entry_count = 5;
```

Comment:
no mixed case in field definitions. Find appropriate snake case definitions for all of these.